# =-=-=-=-=-=-=-=-=-=-=-=-=-=-
# MSeq cohort
# =-=-=-=-=-=-=-=-=-=-=-=-=-=-

#' Multi-region whole-genome sequencing data (median coverage ~100x) for the Set6 sample of the MSeq cohort,
#' originally sequenced at lower resolution by Cross et al. (PMID 30177804).
#' In this new version of the data (doi:10.1101/586560) we provide somatic single nucleotide variants, absolute clonal
#' copy number segments and purity estimates.
#'
#' @docType data
#'
#' @seealso Sample `MSEQ_CRC_ADENOCARCINOMA_SET7`.
#'
#' @usage data(MSEQ_CRC_ADENOCARCINOMA_SET6)
#'
#' @format A list that contains a tibble with mutation, copy number and purity data.
#'
#' @keywords datasets
#'
#' @references G. Caravagna, T. Heide, M. Williams, L. Zapata, D. Nichol, K. Chkhaidze, W. Cross,
#' G.D. Cresswell, B. Werner, A. Acar,  C.P. Barnes,  G. Sanguinetti,  T.A. Graham,  A. Sottoriva.
#' Model-based tumor subclonal reconstruction. biorxiv 2019, doi:10.1101/586560
#'
#' @examples
#' data(MSEQ_CRC_ADENOCARCINOMA_SET6)
#' MSEQ_CRC_ADENOCARCINOMA_SET6
"MSEQ_CRC_ADENOCARCINOMA_SET6"


#' Multi-region whole-genome sequencing data (median coverage ~100x) for the Set7 sample of the MSeq cohort,
#' originally sequenced at lower resolution by Cross et al. (PMID 30177804).
#' In this new version of the data (doi:10.1101/586560) we provide somatic single nucleotide variants, absolute clonal
#' copy number segments and purity estimates.
#'
#' @docType data
#' @seealso Sample `MSEQ_CRC_ADENOCARCINOMA_SET6`.
#' @usage data(MSEQ_CRC_ADENOCARCINOMA_SET7)
#'
#' @format A list that contains a tibble with mutation, copy number and purity data.
#'
#' @keywords datasets
#'
#' @references G. Caravagna, T. Heide, M. Williams, L. Zapata, D. Nichol, K. Chkhaidze, W. Cross,
#' G.D. Cresswell, B. Werner, A. Acar,  C.P. Barnes,  G. Sanguinetti,  T.A. Graham,  A. Sottoriva.
#' Model-based tumor subclonal reconstruction. biorxiv 2019, doi:10.1101/586560
#'
#' @examples
#' data(MSEQ_CRC_ADENOCARCINOMA_SET7)
#' MSEQ_CRC_ADENOCARCINOMA_SET7
"MSEQ_CRC_ADENOCARCINOMA_SET7"

